backend/src/main/java/com/bakdata/conquery/models/query/concept/specific/CQConcept.java

```diff
@@ -135,7 +135,7 @@ private long calculateBitMask(ConceptElement<?>[] concepts) {
 					.toArray(ConceptElement[]::new);
 	}
 
-	private QPNode conceptChild(List<FilterNode<?>> filters, List<QPNode> aggregators) {
+	protected QPNode conceptChild(Concept<?> concept, QueryPlanContext context, List<FilterNode<?>> filters, List<QPNode> aggregators) {
```

Comment:
List<AggregatorNode<?>> aggregartors?
QPNode is zwar das Interface, aber ist das hier nicht zu allgemein für das Argument?

Reply:
Ich kann das umhudeln, aber der braucht hier ne QPNode, weil SpecialDateUnionAggregatorNode ein QPNode ist warum auch immer.